tests/foreman/ui/test_location.py

```diff
@@ -56,6 +56,14 @@ class LocationTestCase(UITestCase):
     """Implements Location tests in UI"""
     location = None
 
+    @classmethod
+    def setUpClass(cls):
+        """Set up an organization for tests."""
+        super(LocationTestCase, cls).setUpClass()
+        cls.org_ = entities.Organization().search(query={
```

Comment:
@rochacbruno @oshtaier How about making a constatnt DEFAULT_ORG_ID and using it in all places, instead of searching?  By this way, we can update the constants file if id changes? :)